lib/y2_installbase.pm

```diff
@@ -297,6 +302,7 @@ sub select_specific_patterns_by_iteration {
         my $selected = check_screen([qw(current-pattern-selected on-category)], 0);
         if ($selected && $selected->{needle}->has_tag('on-category')) {
             move_down;
+            workaround_bsc1189550() if is_sle('>=15-sp3');
```

Comment:
What about 15-SP4, do we know it is impacted in the same way?

Reply:
yes, it reached staging too: https://progress.opensuse.org/issues/97088